packages/runtime/src/prop-controllers/instances.ts

```diff
@@ -120,8 +126,43 @@ class RichTextPropController extends PropController<RichTextPropControllerMessag
   }
 }
 
+export const TableFormFieldsMessageType = {
+  TABLE_FORM_LAYOUT_CHANGE: 'TABLE_FORM_LAYOUT_CHANGE',
+  TABLE_FORM_FIELD_LAYOUT_CHANGE: 'TABLE_FORM_FIELD_LAYOUT_CHANGE',
+} as const
+
+type TableLayoutTableFormFieldsMessage = {
+  type: typeof TableFormFieldsMessageType.TABLE_FORM_LAYOUT_CHANGE
+  value: { id: string; layout: BoxModel }
```

Comment:
We don't have to nest `id` and `layout` inside `value` here

Reply:
Updated

Reply:
That sounds good to me